packages/notification-center/src/store/notifications-provider.context.tsx

```diff
@@ -18,17 +19,15 @@ export function NotificationsProvider({
   children: React.ReactNode;
   stores?: IStore[];
 }) {
-  const firstStore = stores[0];
-  const [storeQuery, setStoreQuery] = useState<IStoreQuery>(() => firstStore.query ?? {});
-  const [storeId, setStoreId] = useState(firstStore.storeId ?? 'default_store');
-  const setStore = useCallback(
-    (newStoreId: string) => {
-      const foundQuery = stores?.find((store) => store.storeId === newStoreId)?.query || {};
-      setStoreId(newStoreId);
-      setStoreQuery(foundQuery);
-    },
-    [stores, setStoreId, setStoreQuery]
+  return (
+    <StoreProvider stores={stores}>
+      <NotificationsProviderInternal>{children}</NotificationsProviderInternal>
+    </StoreProvider>
```

Comment:
basically, I've created the new `StoreProvider` that allows us to use the `storeQuery` state inside of any public hooks like `useUnseenCountQueryKey`... this way we don't require the hook users to pass it through props... it's an internal state info

Reply:
Amazing reason to refactor :)